### PR TITLE
Dash board main display & main navigation

### PR DESCRIPTION
**Pull request by bebegreen**
links to dashboard, github link to the project and logo onclick = () => back to homepage

### CODE REVIEW

**Comment by Alaev** [comment]
@bebegreen please fix your conflicts so we can easy review your changes

File: src/pages/dashboard/DashBoardCard.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  grid-template-areas: 'amount icon' 'subTitle icon';
+  grid-template-rows: 0fr;
+  padding: 1em;
+  box-shadow: 0 0 10px lightgray;
+  margin: 1em;
+  grid-area: ${({ gridArea }) => gridArea};
+  border-left: 5px solid ${({ color }) => color};
+`;
+const DashBoardCard = props => {
+  // console.log(props.children);
```
Comment (andreykishtov) [inline src/pages/dashboard/DashBoardCard.js]: remove console.log

**Comment by bebegreen** [comment]
i dont know how to fix this, it works for me

**Comment by Alaev** [comment]
@bebegreen that is not a good answer :) it work for you as you are working on your local branch that is not synced with master. I will show you tomorrow how to rebase master if you like.

**Comment by Alaev** [comment]
@bebegreen Closing this pr for #15

**Pull request closed without merging**